recipes/core/busybox.yaml

```diff
@@ -30,18 +30,19 @@ buildScript: |
     fi
     makeParallel
     make CONFIG_PREFIX=${BOB_CWD}/install install
+    popd #build
 
     mkdir -p install/.bob
 
     # busybox must be setuid if configured
     grep -qE '^CONFIG_FEATURE_SUID=y$' build/.config && \
-        printf '/usr/bin/busybox f 4755 0  0 - - - - -\n' > install/.bob/busybox.device-table
+       (printf '/usr/bin/busybox f 4755 0  0 - - - - -\n' > install/.bob/busybox.device-table) ||:
```

Comment:
Minor nit: there's no need for a subshell here. But no need to spin another version IMHO.